docs/guide/data_migration.md

```diff
@@ -121,29 +256,41 @@ The optional `--file` option defines the name of the YAML file to export to.
 bin/console ibexa:migrations:generate --type=content --mode=create --file=my_data_export.yaml
 ```
 
+!!! note
+
+    When migrating multiple files at once (for example when calling `ibexa:migration:migrate` without options)
+    they are executed in alphabetical order.
+
 ### user-context
 
 The optional `--user-context` option enables you to run the export command as a specified User.
 The command only exports Repository data that the selected User has access to.
+By default the admin account is used, unless specifically overriden by this option or in
+[bundle configuration](#configuration-reference) (`ibexa_migrations.default_user_login`).
 
 ``` bash
 bin/console ibexa:migrations:generate --type=content --mode=create --user-context=jessica_andaya
 ```
 
-## Importing data
+## Executing migrations
 
-To import Repository data from a YAML file, run the `ibexa:migrations:migrate` command.
+To import Repository data from a YAML files, run the `ibexa:migrations:migrate` command.
```

Comment:
```suggestion
To import Repository data from a YAML file, run the `ibexa:migrations:migrate` command.
```

Reply:
The current version is actually correct. This allows executing multiple files and it's intended to be used this way.